### DISCUSSION TITLE
Release

### DISCUSSION THREAD

**Discussion by surferss** [adepierre/Botcraft#83] [Q&A] [answered]
Can you make a release, please?

**Comment by adepierre** [comment]
Hi!

This is actually not possible for two main reasons:
- depending on your system and what is installed on it, the build can vary. So one build that works on my system would probably not work on yours
- the build requires some minecraft assets to be distributed with, which is forbidden. This assets extraction is automated by the build process

**Comment by adepierre** [comment, marked as answer]
A [release](https://github.com/adepierre/Botcraft/releases/tag/latest) is now available with precompiled versions of the library and binaries for the examples.

Please note that you still have to run a script to download minecraft assets.
